<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MacBook-Pro" ignored="0" name="Default test" tests="12" failures="1" timestamp="26 May 2018 04:22:08 GMT" time="10.759" errors="0">
  <testcase name="R1_enterTextAndVerify" time="0.624" classname="Celmatix.ToDoApp.ToDoMvcTest"/>
  <testcase name="R2_enterTextAndVerify" time="0.208" classname="Celmatix.ToDoApp.ToDoMvcTest"/>
  <testcase name="R3_enterTextAndVerify" time="0.298" classname="Celmatix.ToDoApp.ToDoMvcTest"/>
  <testcase name="R4_countToDoItems" time="0.034" classname="Celmatix.ToDoApp.ToDoMvcTest"/>
  <testcase name="R5_MarkItemCompleted" time="0.128" classname="Celmatix.ToDoApp.ToDoMvcTest"/>
  <testcase name="R6_verifyItemCountAfterTaskComplete" time="0.023" classname="Celmatix.ToDoApp.ToDoMvcTest"/>
  <testcase name="R7_ClearCompletedAndVerifyList" time="0.213" classname="Celmatix.ToDoApp.ToDoMvcTest">
    <failure type="java.lang.AssertionError" message="The completed task “wash cat” is not removed on clearing the completed. expected [false] but found [false]">
      <![CDATA[java.lang.AssertionError: The completed task “wash cat” is not removed on clearing the completed. expected [false] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at Celmatix.ToDoApp.ToDoMvcTest.R7_ClearCompletedAndVerifyList(ToDoMvcTest.java:173)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- R7_ClearCompletedAndVerifyList -->
  <testcase name="R8_VerifyCodeInjectionExploit" time="0.435" classname="Celmatix.ToDoApp.ToDoMvcTest"/>
  <testcase name="R8_VerifyCodeInjectionExploit" time="0.445" classname="Celmatix.ToDoApp.ToDoMvcTest"/>
  <testcase name="R8_VerifyCodeInjectionExploit" time="0.456" classname="Celmatix.ToDoApp.ToDoMvcTest"/>
  <testcase name="R8_VerifyCodeInjectionExploit" time="0.611" classname="Celmatix.ToDoApp.ToDoMvcTest"/>
  <testcase name="R8_VerifyCodeInjectionExploit" time="0.633" classname="Celmatix.ToDoApp.ToDoMvcTest"/>
</testsuite> <!-- Default test -->
